src/bindings/python/src/openvino/runtime/opset15/ops.py

```diff
@@ -144,3 +144,45 @@ def embedding_bag_packed(
         inputs.append(per_sample_weights)
 
     return _get_node_factory_opset15().create("EmbeddingBagPacked", as_nodes(*inputs, name=name), {"reduction": reduction})
+
+
+@nameable_op
+def roi_align_rotated(
+    data: NodeInput,
+    rois: NodeInput,
+    batch_indices: NodeInput,
+    pooled_h: int,
+    pooled_w: int,
+    sampling_ratio: int,
+    spatial_scale: float,
+    clockwise_mode: bool,
```

Comment:
(Side comment, it's not a change request:)
The `clockwise_mode` could be an optional attribute, but
it is actually required by ROIAlignRotated C++ constructor. 
So it's okay to keep it as is, aligned with C++ API.